src/Sylius/Component/Order/Model/Order.php

```diff
@@ -270,8 +258,6 @@ public function removeItem(OrderItemInterface $item)
             $item->setOrder(null);
```

Comment:
These changes are not related to this PR and that change should be discussed separately.